### PR TITLE
Wip/multilanguage

### PR DESCRIPTION
**Pull request by Mehrad0711**
Adding option to have paired sentences as context input (pairs should be semantically-equivalent sentences: translations or paraphrases of each other)

### CODE REVIEW

File: genienlp/tasks/generic_dataset.py [hunk 1]
```diff
@@ -53,11 +53,15 @@ def make_example_id(dataset, example_id):
...
-def same_id(ex):
+def processed_id(ex):
     id_ = ex.example_id.rsplit('/', 1)
     id_ = id_[0] if len(id_) == 1 else id_[1]
+    # translated
     if id_[0] == 'T':
         id_ = id_[1:]
+    # paraphrased
+    if id_[0] == 'P':
+        id_ = id_[1:]
```
Comment (gcampax) [inline genienlp/tasks/generic_dataset.py]: ??? P means PPDB, not paraphrase. Also, why do you care? Stop putting these hacks in genienlp. They don't belong here.

**Review by gcampax — CHANGES_REQUESTED** [review]
More hacks. More options. More "lets add stuff to the library and see what sticks". I'm not happy with this. I'm 100% sure we'll cut out 90% of what you're adding, before the end of the quarter. Just keep this stuff in your branch, alright.

File: genienlp/tasks/generic_dataset.py (hunk 1 above)
Comment (Mehrad0711) [reply genienlp/tasks/generic_dataset.py]: fixed.

**Comment by Mehrad0711** [comment]
[resolved] showing improvement over baseline for translation (2%) and paraphrase (5%) training tasks.

**Pull request merged**